src/Illuminate/Database/Eloquent/Attributes/PrimaryKey.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Illuminate\Database\Eloquent\Attributes;
+
+use Attribute;
+
+#[Attribute(Attribute::TARGET_CLASS)]
+class PrimaryKey
+{
+    /**
+     * Create a new attribute instance.
+     *
+     * @param  string  $name
+     * @param  string  $type
+     * @param  bool  $incrementing
+     * @return void
+     */
+    public function __construct(
+        public string $name = 'id',
```

Comment:
I would avoid specifying the defaults here, since they're already specified as the default values of corresponding properties in `\Illuminate\Database\Eloquent\Model`.

I would use `null` as the default values for this attribute, and only call the corresponding setter if necessary

Reply:
That makes sense, to avoid having to define the defaults in two different places.
I'll update now, thanks for the tip! 💪